spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -235,4 +235,13 @@
       expect(current_path).to eq profile_path
     end
   end
+
+  describe 'clicking the logo image during 2fa process' do
+    it 'signs the user out and returns them to the home page' do
+      user = build_stubbed(:user, :signed_up)
+      sign_in_user(user)
+      find("img[alt='login.gov']").click
+      expect(current_path).to eq root_path
```

Comment:
@monfresh, I actually arrived at this test by starting with the failing code. 

> Or, we might want to allow partially signed in users to access the root url without signing them out, in which case we also want to test that they are not signed out.
To the point at the end of your second comment, the behavior described in the issue on Github was that the user would be signed out when clicking the logo button. If we now want the user to be able to visit the root path in the middle of the 2fa flow (because we changed the content at the root path maybe? I'm not sure why they would want to go back to the sign in page without being signed out), that is a different feature and would be implemented differently.

I do agree that checking for the user's signed out state is something this test should have done.